conf/distro/common-torizon.conf

```diff
@@ -0,0 +1,7 @@
+require conf/distro/include/common-torizon.inc
+require conf/distro/sota.conf.inc
+
+DISTRO = "common-torizon"
+DISTRO_NAME = "Common Torizon OS"
```

Comment:
I think that we should follow the same pattern from upstream something like "Torizon OS Common" instead.

Reply:
@EdTheBearded this does not seem to be resolved. At least one answer like "we decided to follow this decision page ...", "the product manager ask me to follow with this ..." or "we need to decide ..." instead of simple resolve without an answer.

@leograba I did my review and I was ignored for a lot of points. So, next time if someone asks why I'm not involved and would not like to contribute you know why.